<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<<<<<<< HEAD
<testsuite hostname="RahulKaelsMini2" name="Default test" tests="22" failures="0" timestamp="1 Jun 2019 08:53:39 GMT" time="77.983" errors="0">
  <testcase name="launchAndGotoSignUP" time="8.818" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="launchAndGotoSignUP" time="8.818" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="selectPassportTypeTest" time="0.515" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="selectPassportTypeTest" time="0.515" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="selectNationalityTest" time="10.036" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="selectNationalityTest" time="10.036" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="selectPortOfArrivalTest" time="5.334" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="selectPortOfArrivalTest" time="5.334" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="selectVisaType" time="19.468" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="selectVisaType" time="19.468" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="selectVisaLocation" time="5.303" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="selectVisaLocation" time="5.303" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="passPhoneNoTest" time="0.377" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="passPhoneNoTest" time="0.377" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="passEmailIdTest" time="0.357" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="passEmailIdTest" time="0.357" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="passReEmailIdTest" time="0.3" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="passReEmailIdTest" time="0.3" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="enterCaptchaFieldTest" time="4.148" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="enterCaptchaFieldTest" time="4.148" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="selectDateOfBirthTest" time="3.167" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
  <testcase name="selectDateOfBirthTest" time="3.167" classname="com.ghana.app.qa.testcases.SignUpPageTest"/>
=======
<testsuite hostname="Torenzo" name="Default test" tests="4" failures="2" timestamp="28 May 2019 12:29:31 GMT" time="17.492" errors="0">
  <testcase name="verifyVisastatusTab" time="3.188" classname="com.ghana.app.qa.testcases.VisaStatusPageTest"/>
  <testcase name="verifyVisastatusTab" time="3.188" classname="com.ghana.app.qa.testcases.VisaStatusPageTest"/>
  <testcase name="verifyVisastatuswithValidData" time="5.365" classname="com.ghana.app.qa.testcases.VisaStatusPageTest">
    <failure type="java.lang.AssertionError" message="expected [Prasanna Shinde] but found [Prakash Navale]">
      <![CDATA[java.lang.AssertionError: expected [Prasanna Shinde] but found [Prakash Navale]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.ghana.app.qa.testcases.VisaStatusPageTest.verifyVisastatuswithValidData(VisaStatusPageTest.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:603)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyVisastatuswithValidData -->
  <testcase name="verifyVisastatuswithValidData" time="5.365" classname="com.ghana.app.qa.testcases.VisaStatusPageTest">
    <failure type="java.lang.AssertionError" message="expected [Prasanna Shinde] but found [Prakash Navale]">
      <![CDATA[java.lang.AssertionError: expected [Prasanna Shinde] but found [Prakash Navale]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:190)
	at org.testng.Assert.assertEquals(Assert.java:200)
	at com.ghana.app.qa.testcases.VisaStatusPageTest.verifyVisastatuswithValidData(VisaStatusPageTest.java:35)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:749)
	at org.testng.TestRunner.run(TestRunner.java:603)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyVisastatuswithValidData -->
>>>>>>> d7aa21fc9053d7c4ed77e70f387f5e9613b81159
</testsuite> <!-- Default test -->
